mw-webapp/src/service/CurrentProblemService.ts

```diff
@@ -21,4 +22,23 @@ export class CurrentProblemService {
     return currentProblems;
   }
 
+  /**
+   * Read CurrentProblem by Uuid
+   * @returns {Promise<CurrentProblemDTO>} promise of CurrentProblemDTO
```

Comment:
1Dry,
2What does meanread

Reply:
Change to 
  /**
   * Get CurrentProblemDTO
   * @returns {Promise<CurrentProblemDTO>} promise of CurrentProblemDTO
   */

Reply:
I should see it by typescript, meaningless comment